frontend/public/components/monitoring/alerting.tsx

```diff
@@ -585,180 +588,222 @@ const alertStateToProps = (state: RootState, { match }): AlertsDetailsPageProps
   };
```

Comment:
`const alertStateToProps = (state: RootState, { match, location }): AlertsDetailsPageProps `
```suggestion
location,
  };
```